src/app/newsletter/page.tsx

```diff
@@ -3,16 +3,37 @@ import { NewsLetterDataType, NewsLetterSet } from "@/helpers/types";
 import { NewsletterPage } from "../components/server/newsletter";
 import Link from "next/link";
 import { formattedDate } from "@/helpers/utils";
+import { PRODUCTION_URL } from '@/config/config';
+import { getSummaryData } from '../summary/[...path]/page';
 
-// get most recent newsletter from newsletter.json
-const getCurrentNewsletter = () => {
+/** get most recent newsletter from newsletter.json **/
```

Comment:
nit: let's stick one comment style in the code
```suggestion
// get most recent newsletter from newsletter.json
```